client/reader/sidebar/reader-sidebar-nudges/index.jsx

```diff
@@ -75,10 +68,7 @@ function mapStateToProps( state ) {
 			siteCount === 1 && // available when a user owns one site only
 			! isJetpackSite( state, siteId ) && // not for Jetpack sites
```

Comment:
This condition `! isJetpackSite` is already checked as part of `isEligibleForFreeToPaidUpsell`

Reply:
Great sleuthing @tjcafferkey! 

In that case, it seems also that the `isDomainOnlySite` condition is also covered by `isEligibleForFreeToPaidUpsell` by `! siteHasMappedDomain && siteIsOnFreePlan`

Domain only sites do have mapped domains and can't be on a plan (otherwise they wouldn't qualify for "domain only").

3222ecc